### PR TITLE
feat:  add evaluations view with table

### PR DESCRIPTION
**Pull request by ChaiWithJai**

### CODE REVIEW

File: ui/app/templates/evaluations/index.hbs [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+        </button>
+        <button
+          data-test-eval-pagination-next
+          type="button"
+          class="button is-text is-borderless"
+          disabled={{this.shouldDisableNext}}
+          {{on "click" (fn this.onNext @model.meta.nextToken)}}
+        >
+          {{x-icon "chevron-right" class="is-large"}}
+        </button>
```
Comment (jgwhite) [inline ui/app/templates/evaluations/index.hbs:112]: Could we give these buttons a `title` (or `aria-label`)?

**Review by jgwhite — COMMENTED** [review]
The code looks good (only one minor note). Could you give me some steps to test this manually?

File: ui/app/templates/evaluations/index.hbs (hunk 1 above)
Comment (ChaiWithJai) [reply ui/app/templates/evaluations/index.hbs:112]: Where can I learn how to do this properly?

**Review by LevelbossMike — APPROVED** [review]
👏 😍

File: ui/app/adapters/evaluation.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import ApplicationAdapter from './application';
+
+export default class EvaluationAdapater extends ApplicationAdapter {
```
Comment (LevelbossMike) [inline ui/app/adapters/evaluation.js:3]: found a typo - `EvaluationAdapter`

**Pull request merged**